app/src/main/res/values-pt-rBR

```diff
@@ -0,0 +1,401 @@
+    <string name="background_add_wrong_orientation">Plano de Fundo adicionado, mas não visível com a orientação selecionada.</string>
+    <string name="background_add_processing_failed">Falha ao processar a imagem de Plano de Fundo selecionada. Certifique-se de que não está corrompida.</string>
+
+    <string-array name="fast_forward_speed_multiplier_options">
+        <item>Ilimitado</item>
+        <item>1.5x</item>
+        <item>2x</item>
+        <item>3x</item>
+        <item>4x</item>
+        <item>8x</item>
+    </string-array>
+
+    <string-array name="rom_icon_filtering_options">
+        <item>Nenhum (Pixelado)</item>
+        <item>Linear (Suave)</item>
+    </string-array>
+
+    <string-array name="game_runtime_console_type_options">
+        <item>Padrão</item>
+        <item>@string/console_ds</item>
+        <item>@string/console_dsi</item>
+    </string-array>
+
+    <string-array name="firmware_settings_language_options">
+        <item>Inglês</item>
+        <item>Francês</item>
+        <item>Alemão</item>
+        <item>Italiano</item>
+        <item>Espanhol</item>
+        <item>Japonês</item>
+    </string-array>
+
+    <string-array name="game_runtime_mic_source_options">
+        <item>Padrão</item>
+        <item>Nenhum</item>
+        <item>Blow (Sopro)</item>
+        <item>Microfone do dispositivo</item>
+    </string-array>
+
+    <string-array name="console_type_options">
+        <item>@string/console_ds</item>
+        <item>@string/console_dsi</item>
+    </string-array>
+
+    <string-array name="fps_counter_position_options">
+        <item>Oculto</item>
+        <item>Superior esquerdo</item>
+        <item>Centro superior</item>
+        <item>Superior direito</item>
+        <item>Inferior esquerdo</item>
+        <item>Centro inferior</item>
+        <item>Inferior direito</item>
+    </string-array>
+
+    <string-array name="video_filtering_options">
+        <item>Nenhum</item>
+        <item>Linear</item>
+        <item>2xBR</item>
+        <item>HQ2X</item>
+        <item>HQ4X</item>
+        <item>Quilez</item>
+        <item>LCD</item>
+    </string-array>
+
+    <string-array name="audio_interpolation_options">
+        <item>Nenhuma</item>
+        <item>Linear</item>
+        <item>Cosseno</item>
+        <item>Cúbica</item>
+    </string-array>
+
+    <string-array name="audio_bitrate_options">
+        <item>Automático</item>
+        <item>10-bit</item>
+        <item>16-bit</item>
+    </string-array>
+
+    <string-array name="audio_latency_options">
+        <item>Baixa (Lenta)</item>
+        <item>Média</item>
+        <item>Alta (Rápida)</item>
+    </string-array>
+
+    <string-array name="mic_source_options">
+        <item>Nenhum</item>
+        <item>Blow (Sopro)</item>
+        <item>Microfone do dispositivo</item>
+    </string-array>
+
+    <string-array name="save_state_locations">
+        <item>Salvar diretório de arquivo</item>
```

Comment:
Once again, this is supposed to be a name. I think "Diretório de arquivos de salvamento" is a good enough translation.